metafacture-runner/src/main/dist/examples/count/subjects/references_fix.flux

```diff
@@ -0,0 +1,18 @@
+
+default counts=FLUX_DIR + "counts.dat";
+default catalogue = FLUX_DIR + "10.pica";
+
+//count references
+"counting references in " + catalogue | write("stdout");
+
+catalogue
+| open-file
+| as-lines
+| catch-object-exception
+| decode-pica
+| fix(FLUX_DIR + "references.fix")
+| stream-to-triples
+| count-triples(countBy="object")
+| write(counts);
```

Comment:
The Morph example doesn't write to the configured output file (`counts` variable).